Assets/GothicVR/Scripts/World/GameTime.cs

```diff
@@ -10,7 +10,7 @@ namespace GVR.World
     public class GameTime : SingletonBehaviour<GameTime>
     {
         public static readonly DateTime MIN_TIME = new(1, 1, 1, 0, 0, 0);
-        public static readonly DateTime MAX_TIME = new(1, 1, 1, 23, 59, 59);
+        public static readonly DateTime MAX_TIME = new(9999, 12, 31, 23, 59, 59);
```

Comment:
Why? Is it really needed? Because this can cause other issues with GameTime. Yes, it's a DateTime, but used as a Time only right now.

Hm. Let's discuss.

Reply:
It's even better than before. As we need to handle GetDay from Daedalus in the future.